x-pack/plugins/osquery/public/agents/use_agent_policies.ts

```diff
@@ -6,9 +6,10 @@
  */
 
 import { mapKeys } from 'lodash';
-import { useQueries, UseQueryResult } from 'react-query';
+import type { UseQueryResult } from 'react-query';
```

Comment:
Got curious, why this is done in two imports if it could be done in a single one?
```Typescript

import { type UseQueryResult, useQueries } from 'react-query';
```

Is this something can be configured in Eslint?

Same for others with the same behaviour.

Reply:
@dasansol92 
I might be unaware but the rule doesn't allow for such specific auto-formatting. https://typescript-eslint.io/rules/consistent-type-imports/

But if you do import types and values like `import {type X, valueY } from './somefile'` the linter is okay with that. 
So looks like at least for now one has to do this manually for type imports from files that also have value imports.